pkg/assembler/graphql/schema/certifyVuln.graphql

```diff
@@ -71,3 +71,5 @@ extend type Query {
   "Returns all CertifyVuln"
   CertifyVuln(certifyVulnSpec: CertifyVulnSpec): [CertifyVuln!]!
 }
+
+scalar Time
```

Comment:
So there was discussion on using `scalar Time` but we backtracked the change for all the timestamps as we were not sure what the format should look like at this point. We were planning to refine it once we were done with the initial implementation.

Reply:
+1, we were not certain if we were ready to specify the format of this scalar (like `scalar Time @specifiedBy(....)`)

If I understand correctly, Go also uses RFC3339Nano, so at least on that side we won't have issues.

Reply:
Thanks for the feedback. I understand why delaying this decision for later made sense.

The only issue I have is that a default formatting is a bit non-standard and hard to parse outside Go ecosystem. For example:

```
Default: 2023-03-01 10:36:31.630137 +0100 CET m=+0.006376126 // This is how it looks now
RFC3339Nano: 2023-03-01T10:36:31.630137+01:00 // This is how it looks with Time scalar
RFC3339: 2023-03-01T10:36:31+01:00 // IMHO this would be the ideal format
```

I can work for now with my branch to make response parsable with RFC3339Nano. Whenever we decide on the final format, I can pick this up and finish it.

Reply:
Second-level resolution means a lot of events will be bucketed that it would be useful to distinguish. Given a choice I'd prefer the nano variant.

Reply:
Let's go with the Nano variant.

I didn't realize that the formatting gets bad when I was suggesting to use a string instead. It slipped my mind.

Reply:
Great. I agree with @jchestershopify, Nano will make sense in a high-volume system. I'll continue then to change the rest of the API.

Reply:
@mihaimaruseac @pxp928 The PR should now have a fix for the whole API. Let me know if you see anything missing.